tests/testthat/test-utilities.R

```diff
@@ -8,3 +8,21 @@ test_that("It returns the correct name of an enum entry", {
 test_that("It returns NULL if the entry does not exist", {
   expect_equal(netEnumName(enumType = enumType, 5L), NULL)
 })
+
+context("createConfigurationForFile")
+filePath <-getTestDataFilePath("CompiledDataSet.xlsx")
+
+test_that("It can create a DataImporterConfiguration from a XLS file", {
```

Comment:
please deactivate the test under Linux, because now data importer does not work under Linux due to npoi problem
(s. e.g. https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/e9148614cf2085caef1cde0f7a915aaf5f76e954/tests/testthat/test-data-importer-configuration.R#L3-L4 )